### DISCUSSION TITLE
Solidjs.com contents are falling behind new releases

### DISCUSSION THREAD

**Discussion by bobaekang** [solidjs/solid#697] [General]
While it's great to see Solid's new minor releases with exciting and significant(!) features, the official website contents (landing page, guide, docs, tutorial, etc.) seem to be lagging behind the changes.

I was originally going to create an issue on [the `solidjs/solid-docs` repo](https://github.com/solidjs/solid-docs), but then thought it might be worth raising the issue here as a discussion since it might be related to a broader topic of Solid's release process. I know this is much easier said than done, but I believe the release process requires more coordination between the core libraries and the official contents.

**Comment by ryansolid** [comment]
You are right. This is mostly on me. I like moving fast and trust the documents will catch up. It's a huge effort on documentation and @Jutanium is working on it, along with the dozens of people working on translations but I don't usually wait on that stuff. I did for 1.0 which was a necessity, but it also pushed things off weeks. To be fair @Jutanium wanted to meet up this week to talk about the new stuff in 1.2.0 but I pushed it off since I have 2 conference talks and a stream I need to prepare for on top of being an incredibly busy time at work. 

In general, I think we need better information around patch releases. I always give minors a treatment but it would be valuable to generate changelogs for patches as well. I am not as familiar with tooling that automatically does that but I know it exists. I'd like to see docs keep up but I'm hesitant to push back releases for it. I imagine people have different opinions here so I'd love to hear your thoughts.

**Reply by bobaekang** [reply]
I've long been an admirer of your work and productivity, but I get that focus and motivation are big factors there as well.

As for the patch releases, maybe it could be automated--to some extent at least? You probably already know this, but there's a GitHub action to help: https://github.com/marketplace/actions/release-drafter.
